test/Datadog.Trace.ClrProfiler.IntegrationTests/IIS/LoaderOptimizationStartup.cs

```diff
@@ -0,0 +1,58 @@
+#if NETFRAMEWORK
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.IIS
+{
+    public class LoaderOptimizationStartup
+    {
+        private const string Url = "http://localhost:8080";
+
+        public LoaderOptimizationStartup(ITestOutputHelper output)
+        {
+            Output = output;
+        }
+
+        private ITestOutputHelper Output { get; }
+
+        [Fact]
+        [Trait("RunOnWindows", "True")]
+        public async void ApplicationDoesNotReturnErrors()
```

Comment:
You'll want to [avoid `async void`](https://haacked.com/archive/2014/11/11/async-void-methods/) most code.
```suggestion
        public async Task ApplicationDoesNotReturnErrors()
```

Reply:
Whoops, I tried that before and got an error about not all paths returning a value, but it turns out I hadn't included the `System.Threading.Tasks` namespace 😢 I'll fix that